TypeCobol.LanguageServer/JsonRPC/JsonRPCServer.cs

```diff
@@ -278,7 +278,6 @@ private void HandleResponse(string requestId, JToken result, JToken error)
         public void WriteServerLog(string trace)
         {
             messageServer.WriteServerLog(trace);
-            AnalyticsWrapper.Telemetry.TrackEvent(EventType.Diagnostic, trace, LogType.Completion);
```

Comment:
Done in 7aef8b03a1dae17db58ec2fc0a67844457ab6ff9, but there is a trick. In its current state, the `LoggingSystem` is used only with external loggers. So here it is correct to push a trace in the logging system and in the server log file. But in the future when all loggers would be united behind the `LoggingSystem` API, this code will produce duplicate traces in the log file so it will need to be adapted.
I think `StdioHttpServer` won't have a  `logWriter` anymore, instead it will use the logging system without knowing how the traces/exceptions are dispatched.